content/pages/homepage.html

```diff
@@ -112,28 +111,33 @@ <h3 class="ui large monospace header">
 
           <div class="ui centered vertically padded container stackable grid">
 
-            {% call homepage.feature_detail_item(header="Multiple builders", icon="fad fa-terminal") %}
-              Integrated support for Sphinx, MkDocs, and Jupyter Book. Use what works best for your project.
+            {% call homepage.feature_detail_item(header="Supports any doc tool", icon="fad fa-terminal") %}
+              Integrated support for Sphinx, MkDocs, and Jupyter Book.
+              Flexible enough to work with any documentation tool.
             {% endcall %}
 
-            {% call homepage.feature_detail_item(header="Versioned docs", icon="fad fa-layer-group") %}
-              Match your product release cycle with multiple versions of your docs. Automatically build versions from any branch or tag.
+            {% call homepage.feature_detail_item(header="Integrated search", icon="fad fa-search") %}
+              Easily search all your docs via our dashboard,
+              and offer your users a more powerful search with our  search API.
```

Comment:
I'd probably leave dashboard and API out of this description, that's more technical than we need to describe the feature. We describe searching across all projects, which is a good improvement here, we could describe more search features too instead:

```suggestion
              Easily search across all of your projects and
              control search results with fine-grained configuration.
```

Reply:
I was trying to convey both the dashboard & in-doc search. It's really hard to talk about the in-doc search, comparing it against their doc tools search, and API was the best way I could think of to talk about this.

Perhaps it's getting too deep though :)